craft_cli/messages.py

```diff
@@ -42,7 +42,7 @@ class _MessageInfo:
 EmitterMode = enum.Enum("EmitterMode", "QUIET NORMAL VERBOSE TRACE")
 
 # the limit to how many log files to have
-LOG_FILES_LIMIT = 5
+_MAX_LOG_FILES = 5
```

Comment:
I would make this public again and document it so a user of the API is exposed to the default in documentation.

Reply:
As we talked, made everything private.